create_dataframes.py

```diff
@@ -0,0 +1,96 @@
+import listenbrainz_spark
+from listenbrainz_spark import config
+from pyspark.sql import Row, SparkSession
+from datetime import datetime
+import os, sys
+import train_models
+import recommend
+
+def prepare_user_data(table):
+    users_q = spark.sql("""
+            SELECT DISTINCT user_name
+             From %s
+        """ % (table))
+    users_rdd = users_q.rdd.zipWithIndex()
```

Comment:
If there is any way we can do this step as part of the SQL query, that would be ideal.